apps/web/app/api/cron/survey-status/route.ts

```diff
@@ -0,0 +1,70 @@
+import { responses } from "@/app/lib/api/response";
+import { headers } from "next/headers";
+
+import { prisma } from "@formbricks/database";
+import { CRON_SECRET } from "@formbricks/lib/constants";
+
+export async function POST() {
+  const headersList = headers();
+  const apiKey = headersList.get("x-api-key");
+
+  if (!apiKey || apiKey !== CRON_SECRET) {
+    return responses.notAuthenticatedResponse();
+  }
+
+  // close surveys that are in progress and have a closeOnDate in the past
+  const surveys = await prisma.survey.findMany({
```

Comment:
can we rename the `surveys` variable to a more descriptive name, like `surveysToClose`